exercises/space-age/canonical-data.json

```diff
@@ -21,7 +30,16 @@
       "expected": 280.88
     },
     {
-      "description": "age on Venus",
+      "description": "2 Earth years in Mercury years",
+      "property": "age",
+      "input": {
+        "planet": "Mercury",
+        "seconds": 63115200
+      },
+      "expected": 8.3
+    },
+    {
+      "description": "6 Earth years  in Venus years",
```

Comment:
it would probably be good to remove the extra space between `years` and `in`, for the purpose of consistency (no other case has this extra space)